config/config_python.py

```diff
@@ -1,21 +1,22 @@
 import pip
-import pkg_resources
-# *** Add required packages to this list ***
-packages = ['pyyaml', 'numpy', 'matplotlib']
-# *** Add required packages to this list ***
+import sys
+import subprocess
 
-def main(packages, upgrade = False):
-    # Install gslab_tools if it's not installed yet or it has the wrong version
-    pip.main(['install', '--upgrade', 'git+http://git@github.com/gslab-econ/gslab_python.git@v4.1.0'])
 
-    # Install required packages using pip
-    installed_packages = [pkg.key for pkg in pip.get_installed_distributions()]
-    for pkg in packages:
-        if upgrade:
-            pip.main(['install', '--upgrade', pkg])
-        elif pkg not in installed_packages:
-            pip.main(['install', pkg])
+def main(filename='config/requirements.txt', upgrade=False):
+    
+    # Install base python requirements, if they are not installed or are the wrong version
+    command = [sys.executable, '-m', 'pip', 'install', '-r', filename]
+    if upgrade:
+        command.append('--upgrade')
+    subprocess.check_call(command)
+    return None
+
+    # Install gslab_tools
+    subprocess.call(['pip', 'install', "git+https://git@github.com/gslab-econ/gslab_python.git@master"])
```

Comment:
this is part of the function, no? so return None should be below line 16

Reply:
I think I asked you this already, but is there any difference between this line and adding 

`git+https://git@github.com/gslab-econ/gslab_python.git@master`

to `requirements.txt`

Reply:
yes you're right

Reply:
Fixed